### PR TITLE
Turbine better sprites

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Updates the current turbine sprites with 3/4 ones that are so much better. Credit to discord user SnoopCooper#7310

image
image

yes the turbine spins!

## Why It's Good For The Game
Just look at it!

## Changelog

:cl: SnoopCooper#7310, Ghilker (for some minor parts)
imageadd: better turbine sprites
/:cl:

### CODE REVIEW

**Comment by TemporalOroboros** [comment]
Can you show what the sprites are like when it's disassembled? I'm curious how the inside of the turbine is dealt with.

**Comment by Ghilker** [comment]
> Can you show what the sprites are like when it's disassembled? I'm curious how the inside of the turbine is dealt with.

ah sadly there isn't that level of details, it just cuts

**Review by Imaginos16 — APPROVED** [review]
Excellent work!

File: code/modules/power/turbine/turbine.dm [hunk 1]
```diff
@@ -215,6 +240,9 @@
 
 	has_gasmix = TRUE
 
+	active_overlay = "core_light"
```
Comment (LemonInTheDark) [inline code/modules/power/turbine/turbine.dm:243]: you should be using emmissives for this

**Comment by cacogen** [comment]
woah

**Comment by TheBonded** [comment]
we need to remap every station now to show the spinning turbine since it looks like you wont be able to see it when sideways

**Comment by Ghilker** [comment]
> we need to remap every station now to show the spinning turbine since it looks like you wont be able to see it when sideways

true, but i've added a small animation in the exaust side that shows rotation (the black dots moves)

**Comment by notfrying1pans** [comment]
reminds me of goon's science-y machine sprites, i love it so much

**Pull request merged**